src/localization/pageInfoStrings.js

```diff
@@ -28,6 +27,7 @@ export const pageInfoStrings = new LocalizedStrings({
     entered_by: 'Entered By',
     entry_date: 'Entry Date',
     months_stock_required: 'Iraua te Namakaina?',
+    supplier: 'Kambana n oota, Pharmacy ke HC',
```

Comment:
Does `supplier` fit in the table ? and it seems quite long to describe 'supplier'

Reply:
Yes, it is fitting: 
![image](https://user-images.githubusercontent.com/16461988/31523390-af2e5150-b00f-11e7-9851-16b5df479e84.png)

Reply:
Nice!